Node.gitignore

```diff
@@ -79,6 +79,7 @@ web_modules/
 # Next.js build output
 .next
 out
+.env.local
```

Comment:
Understand this is a change requested in Next.js documentation but this should live [here](Note: .env, .env.development, and .env.production files should be included in your repository as they define defaults. .env.*.local should be added to .gitignore, as those files are intended to be ignored. .env.local is where secrets can be stored.)